packages/web-components/fast-foundation/src/tooltip/tooltip.ts

```diff
@@ -290,59 +200,51 @@ export class FASTTooltip extends FASTElement {
     }
 
     /**
-     * invoked when the anchored region's position relative to the anchor changes
+     * Sets the tooltip position
      *
-     * @internal
+     * @public
      */
-    public handlePositionChange = (ev: Event): void => {
-        this.classList.toggle("top", this.region.verticalPosition === "start");
-        this.classList.toggle("bottom", this.region.verticalPosition === "end");
-        this.classList.toggle("inset-top", this.region.verticalPosition === "insetStart");
-        this.classList.toggle(
-            "inset-bottom",
-            this.region.verticalPosition === "insetEnd"
-        );
-        this.classList.toggle(
-            "center-vertical",
-            this.region.verticalPosition === "center"
-        );
-
-        this.classList.toggle("left", this.region.horizontalPosition === "start");
-        this.classList.toggle("right", this.region.horizontalPosition === "end");
-        this.classList.toggle(
-            "inset-left",
-            this.region.horizontalPosition === "insetStart"
-        );
-        this.classList.toggle(
-            "inset-right",
-            this.region.horizontalPosition === "insetEnd"
-        );
-        this.classList.toggle(
-            "center-horizontal",
-            this.region.horizontalPosition === "center"
-        );
-    };
+    public setTooltipPosition = (): void => {
```

Comment:
Is there a mechanism for tooltip contents to style based on tooltip position?  The old version appended classes which authors could use to have a tooltp to the "right" layout differently than a tooltip to the "left"

Reply:
With the removal of `Anchored Region` here, we don't need styling within the shadow root as that's all content coming in as slotted, which is why I dropped these from here. With that said, any position which is passed as an attribute would allow someone to hook to the host element w/ that attribute for styling.

```css
:host([position="top"]) {
    justify-contents: center;
}
 
:host([position="left"]) {
    justify-contents: flex-start;
}
```